zmon-controller-ui/js/controllers/CheckDefinitionCtrl.js

```diff
@@ -18,6 +18,7 @@ angular.module('zmon2App').controller('CheckDefinitionCtrl', ['$scope', '$routeP
         var setLinkToTrialRun = function () {
             if (typeof $scope.checkDefinition === 'undefined') return;
             var params = {
+                id: $scope.checkDefinition.id,
```

Comment:
Can you just rename this to `check_definition_id` or `check_id` (to reduce confusion)?